modules/core/src/main/scala-2/com/caesars/tracing/package.scala

```diff
@@ -0,0 +1,16 @@
+package com.caesars
+
+import io.janstenpickle.trace4cats.inject.EntryPoint
+import io.janstenpickle.trace4cats.kernel.{SpanCompleter, SpanSampler}
+import zio.*
+import zio.interop.catz.*
+
+package object tracing {
```

Comment:
Whats the difference with the scala3 version?

Reply:
Package objects are deprecated in scala 3, I'm unsure when they will be removed entirely from the language.